target/linux/ramips/image/Makefile

```diff
@@ -169,6 +169,29 @@ define Device/Default
   IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | check-size | append-metadata
 endef
 
+define Device/Kroks-mpci
+  DEVICE_PACKAGES := \
+	kmod-usb2 \
+	kmod-usb-acm \
+	kmod-usb-net \
+	kmod-usb-net-cdc-ether \
+	kmod-usb-net-cdc-mbim \
+	kmod-usb-net-cdc-ncm \
+	kmod-usb-net-hso \
+	kmod-usb-net-huawei-cdc-ncm \
+	kmod-usb-net-qmi-wwan \
+	kmod-usb-net-rndis \
+	kmod-usb-net-sierrawireless \
+	kmod-usb-serial \
+	kmod-usb-serial-option \
+	kmod-usb-serial-qualcomm \
+	kmod-usb-serial-sierrawireless \
+	kmod-usb-serial-wwan \
+	kmod-usb-wdm \
+	usb-modeswitch \
+	zram-swap
+endef
```

Comment:
Do we really need all of these for boot?
Packages should be added at the device node.

Reply:
The device supposed to be connected to inet via modem inserted in it's mPCIe slot, only.
So I guess idea behind that is to have it capable for initial inet connection with stock firmware.
We can skip `zram-swap` but I'm not sure about the others modules.

The device comes equipped with Quectel EP-06 modem or some Huaway modem, but optionally might come without the modem at all.

Reply:
Still I wonder what the other similar "modem boards" can suggest here?

For the reference, here is the same device with slightly different "marketing" name:
https://kroksw.com/shop/3g-modems-and-routers/routers-with-sim-injector/router-kroks-rt-ubx-rsim-eq-ep-with-m-pci-modem-lte-cat6-quectel-ep06-e-and-sim-injector/

Reply:
I reduced the packages list to a bare minimum to get LTE connection and thus make the device with no other WAN ports usable out of the box.